src/main/java/dev/binclub/javaception/klass/Klass.java

```diff
@@ -71,4 +101,27 @@ public MethodInfo findMethod(String name, Type[] descriptor) {
 		}
 		return null;
 	}
+	
+	public int getMethodID(String name, String descriptor) {
```

Comment:
In the future this will need to also respect the method ids of superclasses to allow method overriding, but for now maybe we can ignore that

Reply:
method ids are only in context to their own class